### ISSUE TITLE
Targeting 2019-nCoV R&D Discussion 

### ISSUE THREAD

**Issue by lunersonguo** [GHDDI-AILab/Targeting2019-nCoV#4]
**Disclaimer: this discussion is limited within scientific community**

**Comment by YaoWu86** [comment]
Is there any in vitro assay ready for testing of potential inhibitors?

**Comment by Jana66** [comment]
From what we know, there are already in vitro assays ready for screening anti-2019nCoV potency thanks to scientists' overtime work in the Chinese Spring Festival. Several parameters limited these tests' throughput, for the limited Bio-Safety level 3+ labs in China, limited labor in holiday and sample transfer difficulty. These are also some alternative in vitro anti-coronavirus assays, which are suggested for those researchers without Bio-Safety level 3+ labs. (Details seen in reviews from Research Update @ Targeting2019-nCoV) 
At GHDDI, we are actively setting up internal assay platform but it is not ready yet. We will keep you posted.

**Comment by jane-scientist** [comment]
Is there any in vitro assay available for testing of potential inhibitors In GHDDI that we can share now?

**Comment by wingring47** [comment]
> Is there any in vitro assay available for testing of potential inhibitors In GHDDI that we can share now?

Yes, we do. please let us know your specific request here http://ghddionlineform.mikecrm.com/KRLHRTl 

someone from the assay team will contact you shortly. 

Thank you!
